pandas/tests/plotting/test_graphics_others.py

```diff
@@ -18,14 +18,15 @@
 
 import pandas.tools.plotting as plotting
 
-from pandas.tests.test_graphics import (TestPlotBase, _check_plot_works,
-                                        curpath, _ok_for_gaussian_kde)
+from pandas.tests.plotting.common import (TestPlotBase, _check_plot_works,
+                                          curpath, _ok_for_gaussian_kde)
 
 
 """
 These tests are for ``DataFrame.hist``, ``DataFrame.boxplot`` and
 other miscellaneous plots.
-`Dataframe.plot`` and ``Series.plot`` are tested in test_graphics.py
+`Dataframe.plot`` and ``Series.plot`` are tested in
+test_frame.py and test_series.py
```

Comment:
good idea. How about the following structure?

- ``.plot`` (previously ``test_graphics``)
  - ``test_series``
  - ``test_frame``
  - ``test_groupby``

- others (previously ``test_graphics_others``)
  - ``test_legacy_hist``
  - ``test_legacy_boxplot``
  - ``test_misc``

CC: @TomAugspurger